simcon_project/users/views/researcher_home.py

```diff
@@ -1,10 +1,16 @@
 from django.shortcuts import render
 from conversation_templates.models import TemplateResponse
 from django.contrib.auth.decorators import login_required
+from django.core.paginator import Paginator
 
 
 @login_required(login_url="/accounts/login/")
 def ResearcherView(request):
     responseTable = TemplateResponse.objects.all()
```

Comment:
Do we want every researcher to be able to see and delete all responses? Seems like we should filter based on whatever they have assigned. Unrelated to the ticket though

Reply:
we don't have an actual field that relates them back.... i just realized my assumption was wrong about this table only holds there responses